venonactl/cmd/install-agent.go

```diff
@@ -64,6 +98,9 @@ var installAgentCmd = &cobra.Command{
 			CodefreshHost: cfAPIHost,
 		}
 
+		if installAgentCmdOptions.agentToken == "" {
```

Comment:
Why do we need both cfApiToken and agentToken ? those are different tokens. you can't use cfapi token as this is user token vs agent token